HUD/N_0x82cedc33687e1f50.md

```diff
@@ -1,16 +1,18 @@
 ---
 ns: HUD
 ---
-## _0x82CEDC33687E1F50
```

Comment:
When renaming natives, please rename the file and include an alias as well. An example can be found in [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md#guidelines).